### DISCUSSION TITLE
Auth.Users table taking a lot of space

### DISCUSSION THREAD

**Discussion by mtalha101** [supabase/supabase#20423] [Questions] [answered]
I only have 144k users in my auth.users table. The table is already at 211 MB. I have other tables as well reaching up to 200k and 300k rows and taking only a few MBs. Is it related to number of columns in the users table and the type set for each column? Or am I doing something wrong?

<img width="996" alt="Screenshot 2024-01-15 at 2 39 09 AM" src="https://github.com/supabase/supabase/assets/61347696/8887e228-1a10-450f-b582-0b609056776a">

**Comment by GaryAustin1** [comment, marked as answer]
One possibility:
When Postgres updates tables a new row is created and the space from the old row is still allocated to the table.  You need to run some maintenance operations like vacuum.
https://dba.stackexchange.com/questions/187044/disk-space-unreleased-after-cleaning-up-rows-from-pg-table
